src/bootstrap/src/core/sanity.rs

```diff
@@ -177,6 +177,41 @@ than building it.
             continue;
         }
 
+        // Check if there exists a built-in target in the list of supported targets.
+        let cur_dir = env::current_dir().unwrap().display().to_string();
```

Comment:
We should use `build.config.src` to access the project root (because depending on where we invoke `x`, current directory can be different).